### PR TITLE
Add Movie Result Page!

### PR DESCRIPTION
**Pull request by TheGuyWhoCodes**
## Majors
 - Adds a "Movie Results" page
 - Displays meta data for movie
 - Displays three buttons to: write review, add to queue, add to already watched list
     - Shows only if authenticated

## Minors
 - Adds tests
 - Adds static images needed for placeholders.

## Screenshot
image

### CODE REVIEW

**Comment by kirantumkur** [comment]
@XLuna24 @henryksloan  Please review this and provide comments or approve this!

File: mediaphile/src/app/pages/movie-details/movie-details.component.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  }
+
+  public getEntityPosterUrl(): string {
+      if(this.entity["posterPath"]) {
+        return "https://image.tmdb.org/t/p/w500" + this.entity['posterPath']
+      }
+    return "assets/poster-placeholder.png"
+  }
+
+  public addToQueue() {
```
Comment (kirantumkur) [inline mediaphile/src/app/pages/movie-details/movie-details.component.ts:49]: addToQueue**d**?

Comment (TheGuyWhoCodes) [reply mediaphile/src/app/pages/movie-details/movie-details.component.ts]: This sounds a lot better, fixing this for both this function and addToWatched()

Comment (TheGuyWhoCodes) [reply mediaphile/src/app/pages/movie-details/movie-details.component.ts]: fixed with f2acbb8

**Pull request merged**